online-shop/src/app/details/details.component.css

```diff
@@ -0,0 +1,17 @@
+.green {
+  color: green;
+}
+
+.red {
+  color: red;
+}
+h1 {
+  text-align: center;
+  background-color: lightskyblue;
+}
+
+body {
+  background-image: url("https://tse1.mm.bing.net/th/id/OIP.RCalqXxO0ZZxxWBS2QcMqQHaEo?w=256&h=180&c=7&r=0&o=5&dpr=1.25&pid=1.7");
```

Comment:
**suggestion:** again try to have assets already downloaded in your codebase, try not reference them by link. You can add them inside the `/assets` folder